plugins/modules/clusters.py

```diff
@@ -109,6 +110,25 @@ def run_module():
         
         result = dict(changed=True, clusters=[])
 
+    # List events
+    elif module.params.get('action') == "list_events":
+        list_params = {}
+        for k in QUERY_PARAMS_LIST:
+            val = module.params.get(k)
+            if val:
+                list_params = list_params | { k: val }
+        if not module.params.get('cluster_id'):
+            module.fail_json(msg="cluster_id is required for cluster-events action")
+
+        response = requests.get(f"{API_URL}/events", params=list_params, headers=headers)
+        if not response.ok:
+            result = dict(changed=True, response=response.text)
+            module.fail_json(msg=f"Error listing the specified number of events for a given cluster cluster_id: {module.params.get('cluster_id')}", **result)
+        else:
+            result = dict(data=response.json())
+
+        result = dict(changed=True, events=[])
```

Comment:
The change LGTM, but have the same concern as in #23 both PR, provide a different output, I believe it would be better in a different module.

@vjayaramrh toughts?

Reply:
Oh, actually I see @agopala95 implemented the `events` module in #26 I believe it includes this functionality.

Reply:
I am inclined towards keeping a seperate module for events with the changes from #26 and also update it for severity

Reply:
Thanks @pawanpinjarkar, I'll close this then to focus on #26